config/initializers/rollbar.rb

```diff
@@ -0,0 +1,53 @@
+Rollbar.configure do |config|
+  config.access_token = Rails.application.secrets.rollbar_token
+
+  # Without configuration, Rollbar is enabled in all environments.
+  # To disable in specific environments, set config.enabled=false.
+  config.enabled = Rails.env.production? && config.access_token.present?
+
+  # By default, Rollbar will try to call the `current_user` controller method
+  # to fetch the logged-in user object, and then call that object's `id`,
+  # `username`, and `email` methods to fetch those properties. To customize:
+  config.person_method = nil
```

Comment:
For sure, we won't be able to link the ID to an actual user without access to the database (which only sys admins are capable of). But at least we are capturing it and getting the general stats of "hey user 12345 is running into this same error every time they log in? What gives?" Which I am sure at this point if we want to investigate, the sys admins should have no problems with helping us with this?

Also it does seem like "admin" users have access to the users index page at least? https://github.com/ualbertalib/HydraNorth/blob/master/app/controllers/users_controller.rb#L9
Which probably has this information available of what ID corresponds to what user. Maybe this is motivation to do "admin" roles better in ERA?